common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -53,6 +53,12 @@ public abstract class AzureActiveDirectoryAudience {
     public static final String ALL = "common";
     public static final String MSA_MEGA_TENANT_ID = "9188040d-6c67-4c5b-b112-36a304b66dad";
 
+    public static final String MSA_AUDIENCE = "MSA";
+    public static final String COMMON_AUDIENCE = "COMMON";
+    public static final String AAD_AUDIENCE = "AAD";
+    public static final String UNKNOWN_AUDIENCE = "UNKNOWN";
```

Comment:
These are basically the audience values reported in telemetry (rather than the values used to send to eSTS). I suggest we rename each of these constants appropriately such as::

`public static final String MSA_AUDIENCE_FOR_REPORTING = "MSA";`

Alternatively you can consider putting these constants in an inner class for further clarity

```java
public class TelemetryConstants { 
    public static final String MSA_AUDIENCE = "MSA";
    public static final String COMMON_AUDIENCE = "COMMON";
    public static final String AAD_AUDIENCE = "AAD";
    public static final String UNKNOWN_AUDIENCE = "UNKNOWN";
}```

Reply:
[7453c3b](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2495/commits/7453c3bf9cbe16bbc888a59094413c0fb2872d26)